pkg/cloudprovider/cloud.go

```diff
@@ -81,21 +81,21 @@ type LoadBalancer interface {
 	// GetLoadBalancer returns whether the specified load balancer exists, and
 	// if so, what its status is.
 	// Implementations must treat the *api.Service parameter as read-only and not modify it.
```

Comment:
I did document it on each method on the Interface, github just didn't notice properly here.